<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Picaria</class>
 <widget class="QMainWindow" name="Picaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Picaria</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background: url(':/gridRefactor');
	background-repeat: no-repeat;
}</string>
   </property>
   <widget class="Hole" name="hole00">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="Hole" name="hole01">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="Hole" name="hole10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>1</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="Hole" name="hole11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>1</number>
    </property>
    <property name="col" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="Hole" name="hole12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>1</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="Hole" name="hole20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="Hole" name="hole21">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="Hole" name="hole22">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="Hole" name="hole02">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJogo">
    <property name="title">
     <string>Jogo</string>
    </property>
    <addaction name="actionSair"/>
    <addaction name="actionSair_2"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="action13_Buracos"/>
   </widget>
   <widget class="QMenu" name="menuAjuda_2">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre_2"/>
   </widget>
   <addaction name="menuJogo"/>
   <addaction name="menuAjuda"/>
   <addaction name="menuAjuda_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>9 Buracos</string>
   </property>
  </action>
  <action name="actionSair_2">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="action13_Buracos">
   <property name="text">
    <string>13 Buracos</string>
   </property>
  </action>
  <action name="actionSobre_2">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hole</class>
   <extends>QPushButton</extends>
   <header location="global">Hole.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Picaria.qrc"/>
 </resources>
 <connections/>
</ui>
